htdocs/expensereport/class/expensereport_ik.class.php

```diff
@@ -126,6 +126,19 @@ public function fetch($id, $ref = null)
 		return $result;
 	}
 
+		/**
+	 * Load object lines in memory from the database
+	 *
+	 * @return int         <0 if KO, 0 if not found, >0 if OK
+	 */
+	public function fetchLines()
+	{
```

Comment:
there is no table_element_line you don't need to create fetchLines but you can remove the call at line 124